documentation/docs/sidebars.js

```diff
@@ -49,11 +49,11 @@ const sidebars = {
         'atala-prism/overview',
         {
           type: 'category',
-          label: 'PRISM Cloud Agent',
+          label: 'Cloud Agent',
           collapsed: true,
           link: {
             type: 'generated-index',
-            title: 'PRISM Cloud Agent',
+            title: 'Cloud Agent',
```

Comment:
And `Atala PRISM Cloud Agent` to be changed to `Identus Cloud Agent` or `Cloud Agent`